docs/wallet_browser_extension/definition.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - wallet
+  - browser extension
+  - provider API
+---
+
+# Definition
+
+## Dapp
+
+A decentralized application (dapp) is an application built on a decentralized network that combines a smart contract and a frontend user interface.
+
+## Provider
+
+A JavaScript object made available to a consumer, that provides access to wallet by means of a Client.
+
+## Client
+
+An endpoint that receives Remote Procedure Call (RPC) requests from the Provider, and returns their results.
+
+## Wallet
+
+An end-user application that manages private keys, performs signing operations,
+and acts as a middleware between the Provider and the Client.
+
+## Remote Procedure Call (RPC)
+
+A Remote Procedure Call (RPC), is any request submitted to a Provider for some procedure
+that is to be processed by a Provider, its Wallet, or its Client.
+
+## Address
+
+An address is a base64 encoded string representation of a public key.
```

Comment:
Instead of just talk about its type, can describe its actual function? like adding "It is used to identify users in the network." etc.

Reply:
solved by 86f2f28